### PR TITLE
Console.WriteLine -- updating samples for Try .NET

### PR DESCRIPTION
**Pull request by WilliamAntonRohm**
docs PR is https://github.com/dotnet/dotnet-api-docs/pull/3347

### CODE REVIEW

File: snippets/csharp/VS_Snippets_CLR_System/system.Console.WriteLine/CS/WriteLine6.cs [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
    {
+      // <Snippet6>
       Random rnd = new Random();
       // Generate five random Boolean values.
       for (int ctr = 1; ctr <= 5; ctr++) {
          Boolean bln = Convert.ToBoolean(rnd.Next(0, 2));
          Console.WriteLine("True or False: {0}", bln);
       }
+
+      // The example displays the following output:
```
Comment (mairaw) [inline snippets/csharp/VS_Snippets_CLR_System/system.Console.WriteLine/CS/WriteLine6.cs]:
```suggestion
      // The example displays an output similar to the following:
```

File: snippets/csharp/VS_Snippets_CLR_System/system.Console.WriteLine/CS/WriteLine7.cs [hunk 2]
```diff
@@ -1,15 +1,14 @@
-﻿// <Snippet7>
-using System;
+﻿using System;
 
 public class Example
 {
    public static void Main()
    {
+      // <Snippet7>
```
Comment (mairaw) [inline snippets/csharp/VS_Snippets_CLR_System/system.Console.WriteLine/CS/WriteLine7.cs]: let's undo this change since DateTime.Now doesn't pick up the user's local time so it looks a bit weird to have a date different than what you're expecting.

Comment (WilliamAntonRohm) [reply snippets/csharp/VS_Snippets_CLR_System/system.Console.WriteLine/CS/WriteLine7.cs]: Done, thanks.
